utils/src/server/rpc_module.rs

```diff
@@ -370,12 +498,14 @@ impl<Context: Send + Sync + 'static> RpcModule<Context> {
 							send_error(id, &tx, err)
 						}
 					};
+
+					drop(claimed);
```

Comment:
```suggestion
					// Release claimed resources
					drop(claimed);
```

…and add an explicit `drop` for the sync case as well, for readability?